tslint.json

```diff
@@ -1,3 +1,11 @@
 {
-  "extends": "gts/tslint.json"
+  "extends": "gts/tslint.json",
+  "rules": {
+    "only-arrow-functions": false,
+    "no-any": false,
+    "no-unused-expression": false,
+    "variable-name": false,
+    "switch-default": false,
+    "ban": false
```

Comment:
I'm basically overwriting the extended `gts/tslint.json` rules so that we can run the formatter for the time being and to work on reactivating them while I go through this noImplicitAny process